substrate/cli/src/lib.rs

```diff
@@ -280,7 +280,7 @@ where
 			None => 30333,
 		};
 
-		config.network.listen_address = Some(SocketAddr::new("0.0.0.0".parse().unwrap(), port));
+		config.network.listen_address = Some(SocketAddr::new("127.0.0.1".parse().unwrap(), port));
```

Comment:
Why this was changed? I think network should always listen externally.

Reply:
Wuuhooo, sorry, this is a big mistake on my hand, good catch and sorry about that. Will fix.